front/types/assistant/conversation.ts

```diff
@@ -116,7 +116,10 @@ export type AgenticMessageData = {
   originMessageId: string;
 };
 
-export type RichMentionWithStatus = RichMention & { status: MentionStatusType };
+export type RichMentionWithStatus = RichMention & {
+  status: MentionStatusType;
+  canAccessConversation?: boolean; // undefined for approved mentions, boolean for pending
```

Comment:
I probably miss context but isn't it a bit weird to have undefined for approved mentions when the name is `canAccessConversation`? 🤔